internal/config/canary_config.go

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright Strimzi authors.
+// License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+//
+
+// Package config defining the canary configuration parameters
+package config
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+)
+
+// environment variables declaration
+const (
+	BootstrapServersEnvVar = "KAFKA_BOOTSTRAP_SERVERS"
+	TopicEnvVar            = "TOPIC"
+	SendRateEnvVar         = "SEND_RATE"
+	ProducerClientIDEnvVar = "PRODUCER_CLIENT_ID"
+	ConsumerGroupIDEnvVar  = "CONSUMER_GROUP_ID"
+	TLSEnabledEnvVar       = "TLS_ENABLED"
+)
+
+// default values for environment variables
+const (
+	BootstrapServersDefault = "localhost:9092"
+	TopicDefault            = "strimzi-canary"
+	SendRateDefault         = 1000
```

Comment:
Is this a send rate as suggested in the name? Or is this interval between messages?

Reply:
I got your point, it's not a send rate, it's a delay between messages. I will rename it ;-)